### PR TITLE
Update cypress version and fix failing E2E test.

### PR DESCRIPTION
**Pull request by pasindur99**
## Purpose
The purpose of this PR is to update the cypress version and fix failing E2E tests.

## Approach
Update the cypress version.

### CODE REVIEW

**Review by kumuditha-udayanga — APPROVED** [review]

**Comment by kdaud** [comment]
I had disabled all the 3.x tests because they were all failing plans on every push done on the repon so have enabled them.

**Comment by kdaud** [comment]
@pasindur99 does this PR fix the tests?

File: qaframework-bdd-tests/package.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
   "devDependencies": {
     "@testing-library/cypress": "^8.0.0",
     "@types/uuid": "3.4.7",
-    "cypress": "^8.2.0",
+    "cypress": "^9.5.3",
```
Comment (sherrif10) [inline qaframework-bdd-tests/package.json:52]: Is this the latest version of cypress

Comment (pasindur99) [reply qaframework-bdd-tests/package.json:52]: Yeah @sherrif10 this is the stable and latest version of cypress.

**Comment by pasindur99** [comment]
@kdaud  It's totally okay with the disable of E2E 3.X workflows since they all are failing. I did some experiments by updating the cypress version and running the test locally. Some of the tests are running without any error. So if you can enable the workflows to test the PR, it will be easier to identify the bugs on the test.

**Comment by kdaud** [comment]
I enabled them what you need to do is to push a commit on this PR so that the tests run against the PR

**Comment by pasindur99** [comment]
Okay, @kdaud.

**Review by kdaud — APPROVED** [review]

**Comment by kdaud** [comment]
@pasindur99 have merged this PR and would be great if you would follow up on the tests that are still failing in master.

**Pull request merged**